Node.gitignore

```diff
@@ -70,7 +70,7 @@ web_modules/
 
 # dotenv environment variables file
 .env
-.env.test
+.env.*
```

Comment:
This change ignores `.env.example`. So just adding `.env.production` might be better?

Reply:
Yes, I think so.

Reply:
I've also seen  `.env.sample`.  So @kachick please apply the above suggestion.  We only want to ignore `.env.production`.

Reply:
Also, why not add `.env.test`?   Some people may use e.g AWS for testing as well as production and have secret keys for them as well.

Reply:
Thanks for your review! Updated in https://github.com/github/gitignore/pull/3651/commits/b236e1717b90f1755af76900f5387bfa20636de3 !